src/servers/src/http.rs

```diff
@@ -710,7 +711,10 @@ impl HttpServer {
             .layer(
                 ServiceBuilder::new()
                     .layer(HandleErrorLayer::new(handle_error))
-                    .layer(TraceLayer::new_for_http())
+                    .layer(
+                        TraceLayer::new_for_http()
+                            .on_failure(DefaultOnFailure::new().level(Level::DEBUG)),
```

Comment:
Not sure what this change means, may you explain it?

Reply:
ditto